### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andriykooo**

### CODE REVIEW

File: src/arrayMethodSort.js [hunk 1]
```diff
@@ -4,8 +4,34 @@
...
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    if (String(a) > String(b)) {
+      return 1;
+    } else if (String(a) < String(b)) {
+      return -1;
+    } else {
+      return 0;
+    }
+  }) {
```
Comment (Ihor-Karpyn) [inline src/arrayMethodSort.js:15]: Difficult to understand and you can simplify this
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => a.toString() > b.toString()) {
```

File: src/arrayMethodSort.js [hunk 2]
```diff
@@ -4,8 +4,34 @@
...
+    let changes = 0;
+
+    for (let i = 0; i < this.length; i++) {
+      if (compareFunction(this[i], this[i + 1]) > 0 && this[i + 1]) {
```
Comment (Ihor-Karpyn) [inline src/arrayMethodSort.js:19]:
```suggestion
    for (let i = 0; i < this.length - 1; i++) {
      if (compareFunction(this[i], this[i + 1]) > 0) {
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

File: src/arrayMethodSort.js [hunk 3]
```diff
@@ -4,8 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    return a.toString() > b.toString();
+  }) {
```
Comment (Ihor-Karpyn) [inline src/arrayMethodSort.js:9]:
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => a.toString() > b.toString() ) {
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

File: src/arrayMethodSort.js (hunk 3 above)
Comment (Andriykooo) [reply src/arrayMethodSort.js:9]: I cant do like that bcs max length in linter === 80

Comment (Ihor-Karpyn) [reply src/arrayMethodSort.js:9]: Then move to a new line
```suggestion
  [].__proto__.sort2 = function(
  compareFunction = (a, b) =>  a.toString() > b.toString();
  ) {
```

**Review by Ihor-Karpyn — APPROVED** [review]

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**